test/verify/check-subscriptions

```diff
@@ -114,7 +134,7 @@ class TestSubscriptions(MachineCase):
         # enter server and incorrect login data
         b.wait_present("#subscription-register-url-custom")
         b.wait_visible("#subscription-register-url-custom")
-        b.set_val("#subscription-register-url-custom", "{0}:8443/candlepin".format(self.candlepin.address))
+        b.set_val("#subscription-register-url-custom", "{address}:8443/candlepin".format(address=self.candlepin.address))
```

Comment:
While you are at it, making it simpler instead of longer would just be `self.candlepin.address + ":8843/candlepin"`. (Feel free to ignore, of course, it's correct).